docs: add Ricardo Chavarria to GDE resources

ricardochl opened the pull request:
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A


## What is the new behavior?


## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

twerske reviewed (approved):
LGTM - you'll need to rebase for this to be able to run tests and marked for merge

jessicajaniuk reviewed (approved):
LGTM :cookie:

ricardochl committed:
docs: Add Ricardo Chavarria to GDE resources


update size image
update preview
docs: replace an avatar with smaller size
Change order

mary-poppins commented:
You can preview c3a0764 at https://pr43571-c3a0764.ngbuilds.io/.
